kale/embed/mpca.py

```diff
@@ -71,8 +71,8 @@ class MPCA(BaseEstimator, TransformerMixin):
     Args:
         var_ratio (float, optional): Percentage of variance explained (between 0 and 1). Defaults to 0.97.
         max_iter (int, optional): Maximum number of iteration. Defaults to 1.
-        return_vector (bool): Whether ruturn the transformed/projected tensor in vector. Defaults to False.
-        n_components (int): Number of components to keep. Applies only when return_vector=True. Defaults to None.
+        vectorize (bool): Whether ruturn the transformed/projected tensor in vector. Defaults to False.
```

Comment:
Install a spell check in your pycharm/vscode to detect such typos >> ruturn

Reply:
Could you please check my comment above before requesting for a review? Did you fix the typo?